qemu/tests/rh_kselftests_vm.py

```diff
@@ -22,6 +22,10 @@ def run(test, params, env):
     tests_execution_cmd = params.get("tests_execution_cmd")
     whitelist = params.get("whitelist", "").split()
 
+    brew_pkg = params.get("brew_pkg", "brewkoji")
+    if not utils_package.package_install(brew_pkg, session):
+        test.cancel("Install brewkoji package in VM failed")
```

Comment:
Hi @mcasquer 
I think we could make it more robust that we can check if the guest install the essential packages and than install them as a result that we could avoid a installation once.
BTW, I recommend extending the variant package to be a list so that we could be easy to support the other package in the future.
For example:
```
pkgs = brewkoji xxx yyy zzz
------------------------------------------------
for pkg in params.get("pkgs").split():
   if not utils_package.package_install(pkg, session):
```
Please let me know your opinions. Thanks.

Reply:
> BTW, I recommend extending the variant package to be a list so that we could be easy to support the other package in the future.

@YongxueHong that was my thought at the beginning since it's the way package_install is used in other test cases, agree with you :)

Reply:
> I think we could make it more robust that we can check if the guest install the essential packages and than install them as a result that we could avoid a installation once.

@YongxueHong I don't get your point here... :sweat_smile: could you explain it more in deep please?
The current approach looks first for the package with the `rpm -q ...` command. If not installed that means something went wrong in the unattended installation so installs it. What else is needed?

Reply:
@YongxueHong updated to use `params.objects` so when adding more packages they will also get checked and installed if needed

Reply:
> > I think we could make it more robust that we can check if the guest install the essential packages and than install them as a result that we could avoid a installation once.
> 
> @YongxueHong I don't get your point here... 😅 could you explain it more in deep please? The current approach looks first for the package with the `rpm -q ...` command. If not installed that means something went wrong in the unattended installation so installs it. What else is needed?

Hi @mcasquer I checked the `utils_package` module again, I missed the `is_installed` is called within `operate`, so you are right, sorry for inconvenience.

Reply:
> @YongxueHong updated to use `params.objects` so when adding more packages they will also get checked and installed if needed

Hi @mcasquer I suggest that we can use `params.get_list`which is more close to our requirement get a list data type, however the `params.objects` is mainly to return a dict-like object containing the parameters of an individual object, unlike here, we just want to get a list.
Thanks.